concourse/pipelines/checker-bot.yml

```diff
@@ -0,0 +1,71 @@
+---
+roles:
+  - name: owner
+    github:
+      teams: ["haiku:infrastructure"]
+    local:
+      users: ["admin"]
+resource_types:
+- name: gerrit
+  type: registry-image
+  source:
+    repository: docker.io/bluedocks/gerrit-resource
+resources:
+- name: haiku-gerrit
+  type: gerrit
+  source:
+    url: "https://review.haiku-os.org"
+    query: "status:open project:haiku"
+
+# Jobs
+jobs:
+- name: checker-bot
+  public: true
+  plan:
+    - get: haiku-gerrit 
+      trigger: true 
+    - task: check-coding-style
+      config:
+        inputs:
+        - name: haiku-gerrit
+        platform: linux
+        image_resource:
+          type: registry-image
+          source:
+            repository: docker.io/bluedocks/general-worker
```

Comment:
Once #69 is merged, this will be replaced by "haiku/general-worker"

Reply:
good idea to leverage the smaller general-worker here vs the full toolchain container. efficient!